src/player.cpp

```diff
@@ -7200,53 +7222,10 @@ void player::hardcoded_effects(effect &it)
         // a little, and came out of it well into Parched.  Hibernating shouldn't endanger your
         // life like that--but since there's much less fluid reserve than food reserve,
         // simply using the same numbers won't work.
-        if((int(calendar::turn) % 350 == 0) && has_trait("HIBERNATE") && !(hunger > -60) && !(thirst >= 80)) {
```

Comment:
Oh, forgot to mention it in the description.

I integrated it into the other one, because they were nearly identical.